src/app/environment.ts

```diff
@@ -13,17 +13,22 @@ let PROVIDERS: any[] = [
 let _decorateModuleRef = function identity<T>(value: T): T { return value; };
 
 if ('production' === ENV) {
-  // Production
-  disableDebugTools();
   enableProdMode();
+  
+  // Production
+  _decorateModuleRef = (modRef: any) => {
+    disableDebugTools();
+    
+    return modRef;
+  };
 
   PROVIDERS = [
     ...PROVIDERS,
     // custom providers in production
   ];
 
 } else {
-
+  
```

Comment:
@clementprevot - Please kill the whitespace with fire & rebase this with current master so I can get it landed.